src/pkg/tree-list/hooks/useKeyBindings.ts

```diff
@@ -27,11 +29,11 @@ export function useKeyBindings(
       setSelector(selector.dup())
     }
 
-    const bindings = new Mousetrap(ref.current)
+    const bindings = new Mousetrap(node)
```

Comment:
This was a bug where the first time the page loaded, these key bindings got registered globally instead of only local to their parent Dom node.